tasks/shared_libraries/recovery/bin/run_tests.sh

```diff
@@ -18,10 +18,10 @@
 ## Set this for Debugging only
 #set -ex
 
-## Make sure we are calling the script the correct way.
+# Make sure we are calling the script the correct way.
 BASEDIR=$(dirname $0)
-if [ "$BASEDIR" != "." ]; then
-  >&2 echo "ERROR: This script must be called from ."
+if [ "$BASEDIR" != "bin" ]; then
+  >&2 echo "ERROR: This script must be called from [bin/run_tests.sh]."
```

Comment:
Based on your verbiage may want to change `[bin/run_tests.sh]` to `[bin]` or adjust the message string to something like `... from root code directory like the following [bin/run_tests.sh].`

Reply:
changed to  
`>&2 echo "ERROR: This script must be called from [bin]."`